lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -2755,6 +2740,13 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
         adjointGen(adjointGen), builder(getAdjoint()) {}
 
 private:
+  AdjointValue getZeroAdjointValue(SILType type);
```

Comment:
I feel these should be called `make____AdjointValue`, because 1) it conflicts with the semantics of `getAdjointValue`, and 2) it's not _getting_ any intrinsic property from the emitter, but rather creating other data structures based on the emitter's context.

Reply:
Ah, good point. Done in https://github.com/apple/swift/pull/22080/commits/e3c8cb65ccd88c1533e1822f60ae15e60beb0b57